testing/php7-pecl-xlswriter/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: viest <dev@service.viest.me>
+# Maintainer: viest <dev@service.viest.me>
+pkgname="php7-pecl-xlswriter"
+_pkgext=xlswriter
+_pkgreal=php-xlswriter
+pkgver="1.3.1"
+pkgrel=0
+pkgdesc="A PHP Extension for creating and reader XLSX files. "
+url="https://pecl.php.net/package/xlswriter"
+arch="all"
+license="FreeBSD"
```

Comment:
There's no such license https://spdx.org/licenses/
I bet it should be `BSD-2-Clause` https://spdx.org/licenses/BSD-2-Clause.html#licenseText
PS https://github.com/viest/php-ext-xlswriter/commit/8b517d734c1e40e2dd61c26cd45686069bd3c003